src/main/java/blockchains/iaas/uni/stuttgart/de/config/ObjectMapperProvider.java

```diff
@@ -28,6 +33,13 @@ public class ObjectMapperProvider implements ContextResolver<ObjectMapper> {
 
     public ObjectMapperProvider() {
         defaultObjectMapper = createDefaultMapper();
+        if (Boolean.getBoolean("enablePluginsAtStart")) {
```

Comment:
This code checks if discovered plugins should be enabled at the start. If yes, then finds the started plugins and then, find all possible subtypes of `AbstractConnectionProfile` class and register those into ObjectMapper provider. The code is placed here because this should be done during initiation of the provider.